### DISCUSSION TITLE
Prometheus vs InfluxDB

### DISCUSSION THREAD

**Discussion by werenzo** [NetApp/harvest#1140] [Q&A] [answered]
Hello,

I've read docs and some discussions, it seems Prometheus is the preferred choice for datasource. Are there some predefined dashboards for this datasource? Which one exactly?

Are there any predefined dashboard for InfluxDB?

Thank you 

Lorenzo

**Comment by Hardikl** [comment, marked as answer]
Hi @werenzo 

Yes, we do have predefined dashboards for both Prometheus and InfluxDB.

These are for Prometheus: [cmode](https://github.com/NetApp/harvest/tree/main/grafana/dashboards/cmode) and [7mode](https://github.com/NetApp/harvest/tree/main/grafana/dashboards/7mode)
These are for InfluxDB: [influx](https://github.com/NetApp/harvest/tree/main/grafana/dashboards/influxdb)

**Comment by cgrinds** [comment]
And both sets of dashboards can be imported into your Grafana instance by using the `bin/harvest grafana` command. More details [here](https://github.com/NetApp/harvest/blob/main/cmd/tools/grafana/README.md).
